src/main/resources/db/migration/0250/V251__ProjectScores.sql

```diff
@@ -21,4 +21,3 @@ CREATE TABLE accelerator.project_scores (
 
 -- Speed up query by projects
 CREATE INDEX ON accelerator.project_scores(project_id);
```

Comment:
Need to undo the edit to this file; editing migration scripts that have already been applied will cause the server to fail to start.